orders.go

```diff
@@ -395,6 +483,7 @@ func BuildOrderDTO(order shopify.Order) OrderDTO {
 	}
 
 	orderDTO := OrderDTO{
+		Customer:                 BuildCustomerDTO(order.Customer),
```

Comment:
NITPICK: Can this be moved back under `CurrentTotalTaxSet` (where it was originally)?